# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1466097441 25200
#      Thu Jun 16 10:17:21 2016 -0700
# Branch stable
# Node ID 42929c66846dc8aab16a909dcc3eca420447c117
# Parent  2c035963f469d16bd2e855e57739190e6065ea83
doc: delete mention of unsupported syntax for looping over structs (bug #48064)

* stmt.txi (Looping Over Structure Elements): Delete mention of unsupported
syntax for looping over structure elements.

diff --git a/doc/interpreter/stmt.txi b/doc/interpreter/stmt.txi
--- a/doc/interpreter/stmt.txi
+++ b/doc/interpreter/stmt.txi
@@ -653,21 +653,16 @@ endfor
      @print{} val = string
 @end group
 @end example
 
 The elements are not accessed in any particular order.  If you need to
 cycle through the list in a particular way, you will have to use the
 function @code{fieldnames} and sort the list yourself.
 
-The @var{key} variable may also be omitted.  If it is, the brackets are
-also optional.  This is useful for cycling through the values of all the
-structure elements when the names of the elements do not need to be
-known.
-
 @node The break Statement
 @section The break Statement
 @cindex @code{break} statement
 
 The @code{break} statement jumps out of the innermost @code{while},
 @code{do-until}, or @code{for} loop that encloses it.  The @code{break}
 statement may only be used within the body of a loop.  The following
 example finds the smallest divisor of a given integer, and also
